packages/api/Dockerfile.dev

```
@@ -1,9 +1,9 @@
 # run directly from the repo root directory
 # docker build -f ./packages/api/Dockerfile.dev .
-FROM node:20-alpine AS base
+FROM node:20-alpine3.21 AS base
 # =======================================================================
 FROM base AS builder
-RUN apk add --no-cache libc6-compat netcat-openbsd curl
+RUN apk add --no-cache libc6-compat netcat-openbsd curl openssl
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)